diff --git a/data/free worlds checkmate.txt b/data/free worlds checkmate.txt
index a0b2a5568ffb2dee6ba17e237f3a112cc9ebdee6..43684e2f1d7b90d0e04cf6a359bd3d1faddc9725 100644
--- a/data/free worlds checkmate.txt	
+++ b/data/free worlds checkmate.txt	
@@ -1055,6 +1055,7 @@ mission "FWC Checkmate Escorts"
 mission "FWC Pug 1"
 	name "Defend <planet>"
 	description "<planet> is under attack by ships of an unknown type. Drive them off, and then return to the planet to figure out what to do next."
+	source "Silver"
 	to offer
 		has "FWC Checkmate 1C: done"
 	on fail
