diff --git a/data/coalition/coalition missions.txt b/data/coalition/coalition missions.txt
index c1129522299e43fd11c720ca546ab72f7fdc4710..ee2c4e018578614541396918e9ae69b5b6318323 100644
--- a/data/coalition/coalition missions.txt	
+++ b/data/coalition/coalition missions.txt	
@@ -1624,8 +1624,10 @@ mission "Saryd University Lecture"
 				`	"There are many independent worlds that are not ruled by one singular government. Some groups there support piracy, but others are peaceful."`
 					goto pirategood
 			`	"Stamped out, may lawlessness soon be," says the Heliarch in a solemn tone of voice. She picks a Saryd for the next question.`
+				goto questionfour
 			label pirategood
 			`	Your microphone starts to fizzle out as you attempt to answer. The Heliarch's eight eyes are all on you, and you sense that it might not be a good idea to finish your sentence. The Heliarch selects a Saryd for the next question.`
+			label questionfour
 			`	"A fairly comprehensive map of human space, we have." The Saryd brings out a star map of the galactic south. "But of the core, we know not. Inform us of the galaxy, can you?"`
 			choice
 				`	(Tell them about the Korath.)`
