--- a/data/human/deep jobs.txt
+++ b/data/human/deep jobs.txt
@@ -1102,6 +1102,7 @@
 	repeat
 	to offer
 		random < 10
+		has "landing access: Valhalla"
 	source
 		near "Epsilon Leonis" 2 10
 		attributes factory
@@ -1120,6 +1121,7 @@
 	repeat
 	to offer
 		random < 10
+		has "landing access: Valhalla"
 	source
 		near "Epsilon Leonis" 2 10
 		attributes factory research
@@ -1138,6 +1140,7 @@
 	repeat
 	to offer
 		random < 10
+		has "landing access: Valhalla"
 	source
 		near "Epsilon Leonis" 2 10
 		attributes factory mining
