diff --git a/data/transport missions.txt b/data/transport missions.txt
index c65a764702cc30083e4ebec670ab171ade4e0b37..e74911ee39b33bc450707774931d2f39adef64a8 100644
--- a/data/transport missions.txt	
+++ b/data/transport missions.txt	
@@ -2361,6 +2361,7 @@ mission "Terraforming 6"
 mission "Terraforming 7"
 	name "Terraforming Research"
 	description "Bring Amy and Nolan to <destination>, where they will research and publish Amy's terraforming methods."
+	source "Earth"
 	destination "Valhalla"
 	passengers 2
 	blocked "You need <capacity> in order to take on the next mission. Return here when you have the required space free."
